### DISCUSSION TITLE
query for MongoDB via direct entry

### DISCUSSION THREAD

**Discussion by heidebloem** [vercel/next.js#74260] [Help]
### Summary

I am working on my first Next.js project with MongoDB. For testing purposes it is very handy if one can manipulate the database directly through the query box on https://cloud.mongodb.com  on their Collections page.  Can anyone give an example query for entering a character string directly via this query box? This is what I entered myself,  but it leads to an error:
{
  "filter": { "Src": "" },
  " updateOne": { "$set": { "Src": "asdf" } }
}
The same problem when I try it with a find query. What would be the correct query to enter?
 

### Additional information

```js
I entered the query after selecting the correct project and database. I also checked that my Src field is indeed a string type.
I have looked everywhere, but can only find examples for javascript code, but not how to enter a query directly in the tool.
```

**Comment by alexbevi** [comment]
If your question is about inserting documents into a collection via the Atlas UI have a look at https://www.mongodb.com/docs/atlas/tutorial/insert-data-into-your-cluster/ (select "Atlas UI" from the language dropdown). 

Is this what you're looking for? You can edit documents directly from the "Query Results" in the Collections tab via the UI as well.
